packages/backend/src/nest/storage/orbitDb/eventsWithStorage.ts

```diff
@@ -0,0 +1,76 @@
+/**
```

Comment:
This wrapper allows us to use the correct valueEncoding on all of our OrbitDB DBs.